specs/interop/token-bridging.md

```diff
@@ -72,26 +101,200 @@ contract OptimismSuperchainERC20 is ERC20, ISuperchainERC20 {
 }
 ```
 
-## L1 Native Tokens
+[Disclaimer]
+
+- Threat the following code as experimental. It will have errors and will change function names. The goal is to discuss functionalities.
+- Current function names are open to discussion.
+
+## Implementation
+
+### Cross Chain transfers
+
+The standard will include the following three external functions:
+
+- `bridgeERC20(uint256 amount, uint256 chainId)`
+- `bridgeERC20To(address to, uint256 amount, uint256 chainId)`
```

Comment:
Can we add a `sendERC20To(address,uint256,uint256,bytes)` overload per https://github.com/ethereum-optimism/specs/pull/71#issuecomment-1990504944 ? Could this replace the `relayAndExecute`?

Reply:
Yes, we can add this, its a good idea. It should be able to replace `relayAndExecute()` if used with additional logic.